release-notes/v0.11.0.md

```diff
@@ -0,0 +1,55 @@
+I am excited to announce the release of godog v0.11.0-rc1.
+
+Here follows a summary of Notable Changes, the Non Backward Compatible Changes and Deprecation Notices.
+The full change log is available [here](https://github.com/cucumber/godog/blob/master/CHANGELOG.md#v0110-rc1).
+
+
+Notable Changes
+---------------
+
+### Write test report to file
```

Comment:
Might be good to bring to the attention that it doesn't support multiple formatters in the release?

Reply:
could be a good idea, yes